lib/rubocop/cop/rails/file_path.rb

```diff
@@ -29,10 +29,8 @@ class FilePath < Base
         include ConfigurableEnforcedStyle
         include RangeHelp
 
-        MSG_SLASHES = 'Please use `Rails.root.join(\'path/to\')` ' \
-                      'instead.'
-        MSG_ARGUMENTS = 'Please use `Rails.root.join(\'path\', \'to\')` ' \
-                        'instead.'
+        MSG_SLASHES = 'Prefer `Rails.root.join(\'path/to\')`.'
```

Comment:
Aren't most of the other error messages "Prefer x over y"?

Reply:
🤷 better than before ... otherwise I can do `Use`